os/net/netmgr/netdev_mgr_internal.h

```diff
@@ -44,6 +44,7 @@ struct nic_config {
 	int is_default;
 	int loopback;
 	struct nic_io_ops io_ops;
+	struct ifaddrs *addrs;
```

Comment:
We don't know the net so that we can't confirm this is necessary or not. Let's exclude this.

Reply:
I will revert it back